osbuild/buildroot.py

```diff
@@ -157,7 +157,7 @@ def run(self, argv, binds=None, readonly_binds=None):
         # In case you pass `--libdir`, it must be self-contained and we provide
         # nothing else. If you use the system libdir, we additionally look for
         # the installed `osbuild` module and bind-mount it as well.
-        if self._libdir is not None:
+        if not os.listdir("/usr/lib/osbuild"):
```

Comment:
`os.listdir(os.path.join(libdir, "osbuild")` should work! ;)

Reply:
Actually, we need to always bind mount `args.libdir`, i.e.  `mounts += ["--ro-bind", f"{self._libdir}", "/run/osbuild/lib"]` and then do the line 164 to 166 with the `if not os.listdir(os.path.join(libdir, "osbuild"))` condition.
Probably also adjust the comment above.

Reply:
Thank you, made some changes to reflect your suggestions!